symphony-sample/packer-templates/scripts/files/updates.repo

```diff
@@ -0,0 +1,4 @@
+[Updates]
+name=updates
+baseurl="http://mirrors.service.networklayer.com/centos/7/updates/x86_64/"
```

Comment:
This would only work if the build environment is an IBMCloud hosted machine

Reply:
changed to http://mirror.centos.org/centos/7/updates/x86_64/